modules/quotas-and-limits-ibm-power-vs.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// installing/installing_ibm_power/installing-ibm-cloud-account-power-vs.adoc
+
+:_content-type: CONCEPT
+[id="quotas-and-limits-ibm-power-vs_{context}"]
+= Quotas and limits on IBM Power VS
+
+The {product-title} cluster uses a number of IBM Cloud and IBM Power VS components, and the default quotas and limits affect your ability to install {product-title} clusters. If you use certain cluster configurations, deploy your cluster in certain regions, or run multiple clusters from your account, you might need to request additional resources for your IBM Cloud VPC account.
+
+For a comprehensive list of the default IBM Cloud VPC quotas and service limits, see IBM Cloud's documentation for link:https://cloud.ibm.com/docs/vpc?topic=vpc-quotas[Quotas and service limits].
+
+[discrete]
+== Virtual Private Cloud (VPC)
+
+Each {product-title} cluster creates its own VPC. The default quota of VPCs per region is 10. If you have 10 VPCs created, you will need to increase your quota before attempting an installation.
+
+[discrete]
+== Application load balancer
+
+By default, each cluster creates two application load balancers (ALBs):
+
+* Internal load balancer for the master API server
+* External load balancer for the master API server
+
+You can create additional `LoadBalancer` service objects to create additional ALBs. The default quota of VPC ALBs are 50 per region. To have more than 50 ALBs, you must increase this quota.
+
+VPC ALBs are supported. Classic ALBs are not supported for IBM Power VS.
+
+[discrete]
+== Cloud connections
+
+There is a limit of two cloud connections per Power VS instance. It is recommended that you have only one cloud connection in your Power VS instance to serve your cluster.
+
+[discrete]
+== DHCP Service
+
+There is a limit of one DHCP service per Power VS instance.
+
+[discrete]
+== Networking
+
+Due to networking limitations, there is a restriction of one OpenShift cluster installed via IPI per zone per account. This is not configurable.
+
+[discrete]
+== Virtual Server Instances (VSI)
+
+By default, a cluster creates server instances with the following resources :
+
+* 0.5 CPUs
+* 32 GB RAM
+* System Type: s922
+* Processor Type: uncapped, shared
+* Storage Tier: Tier-3
+
+The following nodes are created:
+
+* One bootstrap machine, which is removed after the installation is complete
+* Three control plane nodes
+* Three compute nodes
+
+For more information, see IBM Power VS's documentation on instance options link:https://cloud.ibm.com/docs/vpc?topic=vpc-profiles[Creating a Power Systems Virtual Server].
```

Comment:
For more information, see link:https://cloud.ibm.com/docs/vpc?topic=vpc-profiles[Instance profiles] in the IBM Cloud documentation.

Reply:
@alishaIBM can you double-check the google doc, and let me know if there's no link to Power VS vms there?

Reply:
yes, thanks @SNiemann15